src/ui/hooks/comments/useAddComment.tsx

```diff
@@ -19,7 +19,8 @@ export default function useAddComment() {
           }
         }
       }
-    `
+    `,
+    { refetchQueries: ["GetComments"] }
```

Comment:
When we save a new comment, we should refetch the list of comments too (so the comment doesn't disappear).